wordle-web/src/components/NavBar.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <nav>
+    <v-toolbar color="primary" flat app>
+      <v-toolbar-title @click="$router.push('/')"
+        ><v-icon>mdi-diamond-stone</v-icon> Wordle Redux</v-toolbar-title
+      >
+      <v-spacer></v-spacer>
+      <v-toolbar-items class="red--text">
+        <v-btn to="/">Home</v-btn>
+        <v-btn to="/wordle">Wordle</v-btn>
+        <v-btn to="/about">About</v-btn>
+        <v-app-bar-nav-icon @click="setDefault" icon="mdi-theme-light-dark"> </v-app-bar-nav-icon>
+        <v-app-bar-nav-icon @click.stop="setting = true" icon="mdi-cog"></v-app-bar-nav-icon>
+        <v-app-bar-nav-icon @click.stop="drawer = !drawer"></v-app-bar-nav-icon>
+      </v-toolbar-items>
+    </v-toolbar>
+
+    <v-navigation-drawer location="right" @click.stop="drawer = !drawer" v-model="drawer" temporary>
+      <v-list-item>
+        <v-list-item-content>
+          <v-list-item-title class="title">Wordle Redux</v-list-item-title>
+          <v-list-item-subtitle>Wordle Redux</v-list-item-subtitle>
+          <v-list-item @click.stop="drawer = !drawer" prepend-icon="mdi-home" to="/">
+            Home
+          </v-list-item>
+          <v-list-item @click.stop="drawer = !drawer" prepend-icon="mdi-star" to="/wordle">
+            Wordle
+          </v-list-item>
+          <v-list-item @click.stop="drawer = !drawer" prepend-icon="mdi-information" to="/about">
+            About</v-list-item
+          >
+        </v-list-item-content>
+      </v-list-item>
+    </v-navigation-drawer>
+  </nav>
+  <v-dialog
+        v-model="setting"
+        width="auto"
+      >
+      <v-card>
+      <v-list-item>
```

Comment:
I think you could reduce alot of this code by using a typescript object that you can loop on. That'll also let you have a single function for changing the theme, rather than one function per-theme